inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -685,7 +690,8 @@ int main(int argc, char* argv[]) {
         std::cout << "Count:      " << iteration << " iterations" << std::endl;
         std::cout << "Duration:   " << double_to_string(totalDuration) << " ms" << std::endl;
         if (device_name.find("MULTI") == std::string::npos)
-            std::cout << "Latency:    " << double_to_string(latency) << " ms" << std::endl;
+            std::cout << "Latency (" << FLAGS_latency_percentile << " percentile):    "
+                      << double_to_string(latency) << " ms" << std::endl;
```

Comment:
Python benchamark_app should be updated accordingly

Reply:
Where I can find this benchmark_app for python?

Reply:
Please look here https://github.com/openvinotoolkit/openvino/tree/master/tools/benchmark